src/components/Observations/TopCard.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import type { Node } from "react";
+import React from "react";
+import { View } from "react-native";
+
+import useLoggedIn from "../../sharedHooks/useLoggedIn";
+import { viewStyles } from "../../styles/observations/topCard";
+import LoggedOutCard from "./LoggedOutCard";
+import UserCard from "./UserCard";
+
+type Props = {
+  numObsToUpload: number
+}
+
+const TopCard = ( { numObsToUpload }: Props ): Node => {
+  const isLoggedIn = useLoggedIn( );
+  if ( isLoggedIn === null ) {
```

Comment:
What does it mean when `isLoggedIn` is null? I think this is definitely an improvement in that I'm not seeing the `LoggedOutCard` flash before seeing the `UserCard` when I'm logged in, but now there seem to be three potential views that could be rendered, which still results in some visible jitter in height. Maybe it would be better to render them all as children of as single component that always gets rendered and has a fixed height?